back/app/models/admin_publication.rb

```diff
@@ -29,11 +29,12 @@
 class AdminPublication < ApplicationRecord
   PUBLICATION_STATUSES = %w[draft published archived]
 
+  belongs_to :publication, polymorphic: true, touch: true
```

Comment:
I think it makes sense to move the associations on top of the files, so that I as a reader see:

- what associations does the model have

and then

- what kind of validations (on these associations) exist